docs/dev/setup.rst

```diff
@@ -342,18 +339,18 @@ Other useful commands:
 -  Stop the client: ``docker-compose stop artemis-client`` (restart via
    ``docker-compose start artemis-client``)
 
-Text Assessment Clustering Service
+Athene Service
 ----------------------------------
```

Comment:
This should be trimmed to the headline length to be consistent with the rest of our docs:
```suggestion
Athene Service
--------------
```